.github/workflows/ci.yml

```diff
@@ -120,6 +120,18 @@ jobs:
           bazel test //tensorboard/summary/writer:event_file_writer_s3_test &&
           bazel test //tensorboard/compat/tensorflow_stub:gfile_fsspec_test &&
           bazel test //tensorboard/summary/writer:event_file_writer_fsspec_test
+      - name: 'Bazel: build Pip package (with TensorFlow only)'
+        if: matrix.tf_version_id == 'tf'
+        run: |
+          ./tensorboard/tools/mirror_urls_test.sh
```

Comment:
Should be update_version.sh?

Reply:
oops, forgot to commit the latest change, thanks!

Reply:
Also had to add `chmod a+x` for the script.